### DISCUSSION TITLE
Конструктор напоминалок из фильтра.

### DISCUSSION THREAD

**Discussion by savosik** [moonshine-software/moonshine#1147] [Ideas]
Пример: пользователю нужно вывести напоминание, что заявка просрочена, близиться дедлайн, или заказ перешел в новый статус.  Можно добавить в фильтр рядом с кнопками Поиск, сбросить, ещё кнопку "создать напоминание", сохранив при этом фильтр как триггер для получения notification, sweetalert или письма на email.  
![Screenshot_2024-07-30-21-59-27-617_com android chrome](https://github.com/user-attachments/assets/a7a7c4ef-1e41-4b43-81da-769f8a0652e1)

**Comment by lee-to** [comment]
Какой то личный кейс, не претендующий на попадание в коробку универсального решения, максимум как отдельный пакет который на данный момент команде муншайн не интересен
В любом случае спасибо за идею но вы должны понимать что решения в коробке должны быть популярные и нужны большинству
Также мы даем все возможности чтобы реализовать подобное самостоятельно, для этого нужно просто быть разработчиком и основная концепция муншайн давать такие возможности разработчикам и делать их счастливее
